Automattic/vip-go-mu-plugins
# Updating Parse.ly 3.1 branch to 3.1.3

## Description

This PR pulls in the latest minor release of the Parse.ly plugin.

## Changelog Description

### Plugin Updated: Parse.ly 3.1.3

We upgraded wp-parsely from 3.1.2 to 3.1.3

## Checklist

Please make sure the items below have been covered before requesting a review:


## Steps to Test

1. Sandbox a test VIP Go site to test against (e.g. https://blog.parse.ly)
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
    * You should see version `3.1.2` at this point and meta data and tracking script should be loaded as before
1. Copy modified files from this branch to the appropriate location on the sandbox (e.g. the `wp-content/mu-plugins/wp-parsely-3.1` directory)
1. Confirm current status (what plugin version is loading, are meta data printed, is tracking script loaded, etc.)
    * You should see version `3.1.3` at this point and meta data and tracking script should be loaded as before

No new errors or warnings should be apparent.
